cypress/e2e/createappointment.cy.ts

```diff
@@ -15,31 +15,46 @@ describe('Create New Appointment', () => {
         cy.contains('Appointments').should('exist');
     });
     it('create new appointment', () => {
-        cy.contains('New Appointment').should('exist').click();
-        cy.get('#demo-simple-select').click().get('li.MuiButtonBase-root:nth-child(10)').click();
-        cy.get('input#title').should('exist').type('help with laundry');
-        cy.get('input#location').should('exist').type('1123 star ave, Richmond');
-        cy.get('button.MuiButtonBase-root:nth-child(1) > svg:nth-child(1)')
-            .click()
-            .get('div.MuiDayPicker-weekContainer:nth-child(2) > button:nth-child(6)')
+        cy.contains('New Appointment').should('exist').click({ multiple: true });
+        cy.get('#demo-simple-select')
             .click()
-            .get('div.MuiPaper-root')
-            .click()
-            .get('.MuiClock-squareMask')
-            .click();
-        // cy.wait(2000);
-        cy.contains('button', 'Schedule').should('exist').click();
+            .get('li.MuiButtonBase-root:nth-child(10)')
+            .click({ multiple: true });
+        const timeNow = new Date().getTime();
+        cy.get('input#title').should('exist').type('help with groceries ').type(timeNow.toString());
+        cy.get('input#location').should('exist').type('1123 star ave, Richmond');
+        cy.get('button.MuiButtonBase-root:nth-child(1) > svg:nth-child(1)');
+        cy.get('input[placeholder="mm/dd/yyyy hh:mm (a|p)m"]')
+            .should('exist')
+            .clear()
+            .type('11/29/2022 03:00 PM');
+
+        cy.contains('button', 'Schedule').should('exist').click({ multiple: true });
         cy.contains('Appointments').should('exist');
-        cy.wait(2000);
+
         // Check if the appointment details are displayed on the list
-        cy.contains('help with laundry').should('exist');
+        cy.contains('help with gardening').should('exist');
```

Comment:
Is this working? the appointment titles are inconsistent. You are typing "help with groceries" above and now you have "help with gardening"

Reply:
Can we extract this one into a reusable constant